README.mkd

```diff
@@ -279,7 +281,7 @@ Vendors should also aim to follow the conventions used for naming mnemonics
 in the ratified base ISA and extensions (e.g. the use of 'w', 'd',
 'u', and 's' suffixes).
 
-The vendor suffix requires at least two letters.
+The vendor prefix requires at least two letters.
 
 NOTE: Although no centralized registration is required, vendors should add the
```

Comment:
This is also about prefixes, so if the other paragraph is moved then this one probably should also move.

Reply:
> Should we also disallow [Zz] as the first letter of the vendor-prefix, as standard-extensions are names Z?

I am less worry about `Z*`, but I saw `cm.` prefixed are used in 

> Should we state somewhere that the vendor-prefix is allocated as case-insensitive?

Yeah, thanks for remind that, GNU assembly and LLVM assembly are both case-insensitive on the opcode parsing.

Reply:
> This is also about prefixes, so if the other paragraph is moved then this one probably should also move.

Moved